lib/main.js

```diff
@@ -25,9 +25,9 @@ export const allImplementations = implementations;
  * @param {Map} [options.implementations=allImplementations] - A Map of
  *   implementations.
  * @param {Function<boolean>} options.filter - A function to
- * filter the map's entries on that returns true or false.
+ *   filter the map's entries on that returns true or false.
```

Comment:
This is a guess at what this description is meant to say. If I got it wrong, that's just evidence that it needed revision.
```suggestion
 *   filter the map's entries, that returns true or false.
```